src/TabContent.js

```diff
@@ -34,7 +34,7 @@ export default class TabContent extends Component {
     const classes = classnames('tab-content', this.props.className);
     return (
       <div className={classes}>
-        { this.props.children }
```

Comment:
Had to make the linter happy in order to run test locally. Not sure why no one else is getting this (including Travis) when running the tests.